src/django/contricleaner/lib/serializers/row_serializers/row_required_fields_serializer.py

```diff
@@ -0,0 +1,20 @@
+from .row_serializer import RowSerializer
+
+
+class RowRequiredFieldsSerializer(RowSerializer):
```

Comment:
I think. there's a better implementation could be copied from prev closed PR:

```
class HeaderRequredValidator:
     required_fields = {"name", "address", ""}

     def validate(self, raw_header: List[str]) -> dict:
         diff = self.required_fields.difference(raw_header)

         if len(diff) > 0:
             return {
                 "error": {
                     "message": "{} are missing".format(', '.join(diff)),
                     "type": "Error",
                 },
             }
         return {}
```

Reply:
Will do